tasks/fix-cat2.yml

```diff
@@ -485,7 +485,9 @@
       - pamd
 
 - name: "MEDIUM | RHEL-07-010340 | PATCH | Users must provide a password for privilege escalation."
```

Comment:
if we're not actually fixing the issue, we probably just want to call it "AUDIT"